modules/runtime/nuxeo-common/src/main/java/org/nuxeo/common/xmap/registry/Registry.java

```diff
@@ -32,6 +32,21 @@
      */
     boolean isNull();
 
+    /**
+     * Marks the registry with given identifier.
```

Comment:
You should mention that a registry can have not just zero or one but also more than one.